02-source_channel/README.md

```diff
@@ -0,0 +1,217 @@
+# Source to Channel with multiple Subscribers 
+
+A more advanced and flexible use case of Knative eventing is to sink the events from the emitting source to a Knative Eventing channel. On the channel we can have multiple subscribers, that are all receiving the _same_ events, for different purposes. 
```

Comment:
"that are all receiving the _same_ events": tbh i don't get if you mean broadcasting events or "at most one receiver receives a particular event instance", but it's my ignorance

Reply:
I mean broadcasting